omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -780,9 +780,13 @@ def migrate_keys
       credentials.save if did_something
     end
 
+    def set_ldap_enabled
+      PrivateChef["ldap"]["enabled"] = !( PrivateChef["ldap"].nil? || PrivateChef["ldap"].empty? )
```

Comment:
Is one of these checks maybe redundant? I'm wondering because _if `PrivateChef["ldap"].nil?` was true_, wouldn't `PrivateChef["ldap"]["enabled"] = ...` fail with `[]=` not defined for NilClass? 🤔